File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/EnemyHextechDragonKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class EnemyHextechDragonKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playEnemyHextechDragonKillAnimation() {
        new AnimationTester().testAnimation(new EnemyHextechDragonKillAnimation());
    }
}

File: test/hud/animations/EnemyHextechDragonKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {EnemyHextechDragonKillAnimation} from '../../../src/hud/animations/EnemyHextechDragonKillAnimation.js';

/**
 * Integration test for enemy hextech dragon kill animation on actual hardware.
 */
describe('EnemyHextechDragonKillAnimation Integration', () => {
    test('playEnemyHextechDragonKillAnimation', async () => {
        await new AnimationTester().testAnimation(new EnemyHextechDragonKillAnimation());
    }, 30000);
});
